app/models/finance/statement.rb

```diff
@@ -7,7 +7,10 @@ class Finance::Statement < ApplicationRecord
 
   belongs_to :cpd_lead_provider
   belongs_to :cohort
+
   has_many :participant_declarations
+  has_many :statement_line_items, class_name: "Finance::StatementLineItem"
```

Comment:
Note for later: we'll need to apply the correct scope to the subclasses. So that we can query and aggregate on those on the payment breakdowns